src/wish.cpp

```diff
@@ -170,7 +170,9 @@ class wish_mutate_callback: public uimenu_callback
 
             mvwprintz( menu->window, menu->w_height - 3, startx, c_green, "%s", msg.c_str() );
             msg = padding;
-            mvwprintw( menu->window, menu->w_height - 2, startx, _( "[/] find, [q]uit" ) );
+            input_context ctxt( "UIMENU" );
+            mvwprintw( menu->window, menu->w_height - 2, startx, string_format( _( "[%s] find, [%s] quit" ),
```

Comment:
`mvwprintw` does formatting, no need to invoke `string_format`. Actually, this can cause trouble if the formatted string contains a '%', which is interpreted by `mvwprintw` and expects another (missing) argument.